_includes/install-script.md

```diff
@@ -0,0 +1,62 @@
+<!-- This file is included in Docker CE or EE installation docs for Linux. -->
```

Comment:
This could also use a link to the github repo as well if there are issues / changes that the community would like to make. https://github.com/docker/docker-install